cves/CVE-2013-0863.yml

```diff
@@ -186,8 +192,9 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: true
+  answer: Proper unit testing could have caught this. There should have been some kind of check 
```

Comment:
While unit testing could have caught this, I think this is in reference to the attacker's side of things, meaning fuzzers and the like.